package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "webauth-i-challenge",
+  "version": "1.0.0",
+  "description": "## Topics",
+  "main": "index.js",
+  "dependencies": {
+    "connect-session-knex": "^1.4.0",
+    "express-session": "^1.16.1"
+  },
+  "devDependencies": {
+    "bcryptjs": "^2.4.3",
```

Comment:
You'll need to remove all these except node on from the dev dependency section and add them as regular dependencies